Packs/PhishUp/Playbooks/PhishUp_Mail_Scanner.yml

```diff
@@ -0,0 +1,790 @@
+      version: -1
+      name: PhishUp Actions
+      type: title
+      iscommand: false
+      brand: ""
+      description: ""
+    nexttasks:
+      '#none#':
+      - "15"
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": -430,
+          "y": 880
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "8":
+    id: "8"
+    taskid: 9774d2de-5f2b-4d2f-8b26-931d985e88cc
+    type: condition
+    task:
+      id: 9774d2de-5f2b-4d2f-8b26-931d985e88cc
+      version: -1
+      name: Detect Mail's Source Brand
+      description: Detect Mail's Source Brand
+      type: condition
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#default#':
+      - "19"
+      Gmail:
+      - "16"
+    separatecontext: false
+    conditions:
+    - label: Gmail
+      condition:
+      - - operator: isEqualString
+          left:
+            value:
+              simple: incident.sourceBrand
+            iscontext: true
+          right:
+            value:
+              simple: Gmail
+    view: |-
+      {
+        "position": {
+          "x": -730,
+          "y": 1360
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "9":
+    id: "9"
+    taskid: 344f1e3c-19fe-47ef-8dca-cd8ced179d65
+    type: regular
+    task:
+      id: 344f1e3c-19fe-47ef-8dca-cd8ced179d65
+      version: -1
+      name: Gmail Delete Mail
+      description: Deletes an email in the user's mailbox.
+      script: Gmail|||gmail-delete-mail
+      type: regular
+      iscommand: true
+      brand: Gmail
+    nexttasks:
+      '#none#':
+      - "19"
+    scriptarguments:
+      message-id:
+        simple: ${incident.emailmessageid}
+      permanent:
+        simple: "False"
+      user-id:
+        simple: ${incident.emailto}
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": -1030,
+          "y": 1850
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "12":
+    id: "12"
+    taskid: 520cb2df-1a4b-4864-811c-49d5a90512be
+    type: condition
+    task:
+      id: 520cb2df-1a4b-4864-811c-49d5a90512be
+      version: -1
+      name: Is Any URL found?
+      description: Checking if mail has any url in body?
+      type: condition
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#default#':
+      - "2"
+      No URL:
+      - "24"
+    separatecontext: false
+    conditions:
+    - label: No URL
+      condition:
+      - - operator: isEmpty
+          left:
+            value:
+              simple: ExtractedIndicators
```

Comment:
Should be `ExtractedIndicators.URL` since other indicators other than URL may be found.